aten/src/ATen/Declarations.cwrap

```diff
@@ -1084,7 +1084,6 @@
     - Int
     - Short
   backends:
-    - CPU
```

Comment:
What does it mean to remove the CPU backend from cwrap? (Not a leading question; I really don't know!)

Reply:
At a high level, it means it's not possible to call the TH version of this anymore from ATen, so it'll need to come from ATen itself.